ui/src/treeDataProvider/AzureTreeDataProvider.ts

```diff
@@ -111,8 +115,10 @@ export class AzureTreeDataProvider implements TreeDataProvider<IAzureNode>, Disp
                         return new SubscriptionNode(this, this._resourceProvider, subscriptionInfo.subscription.subscriptionId, subscriptionInfo.subscription.displayName, subscriptionInfo);
                     }
                 });
-                return this._subscriptionNodes;
+                nodes = this._subscriptionNodes;
             }
+
+            return nodes.concat(this._rootNodes);
```

Comment:
should root nodes come first or last in the list?

Reply:
I chose last to be consistent with current behavior (The 'Attached Mongo Servers' node comes after subscriptions in cosmos db).

I think subscriptions take priority for an 'Azure Tree Provider'